### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by DoubleOTheven**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -715,7 +715,6 @@ Specifically you need to upgrade to at least the pallet version
 ### Added
 - Added support for wildcard selectors ‒ [#1020](https://github.com/paritytech/ink/pull/1020).
     - This enables writing upgradeable smart contracts using the proxy/forward pattern.
-      We added a new example to illustrate this ‒ the [proxy](https://github.com/paritytech/ink/tree/master/examples/proxy) example.
```
Comment (DoubleOTheven) [inline CHANGELOG.md:718]: I removed this because I did not see a proxy example. Does one exist?

**Comment by TomaszWaszczyk** [comment]
Here are also dead links. 

I promise not looking for more dead links. :D


https://user-images.githubusercontent.com/1605705/228076859-2c357707-ce4c-4359-8219-fa4b8be25b91.mp4

**Comment by DoubleOTheven** [comment]
> Can you check for broken links in codebase as well plz?

Done.

**Review by SkymanOne — APPROVED** [review]
Thanks!

**Pull request merged**